csharp/sdk/Maple/Dtmf.cs

```diff
@@ -8,16 +9,35 @@ namespace Maple
 {
     class Dtmf
     {
+        private const float DEFAULT_MIC_LEVEL = 0.7f;
         private readonly Double DEFAULT_GAIN = 0.6;
         private readonly TimeSpan DEFAULT_TONE_DURATION_MS = TimeSpan.FromMilliseconds(100);
         private readonly TimeSpan DEFAULT_TONE_PAUSE_DURATION_MS = TimeSpan.FromMilliseconds(100);
+        private readonly TimeSpan DEFAULT_TONE_EXIT_PAUSE_MS = TimeSpan.FromMilliseconds(300);
+        private float lastMicLevel;
 
         private Dictionary<char, Tuple<int, int>> DtmfLookup;
 
         public Dtmf()
         {
         }
 
+        private void MuteMic()
+        {
+            var mic = new MMDeviceEnumerator().GetDefaultAudioEndpoint(DataFlow.Capture, Role.Communications);
```

Comment:
@lukebayes we utilize the default microphone for things like voice access. This could cause misbehavior when muting. Please consider muting the default microphone instead if this is possible with NAudio.